src/index.css

```diff
@@ -0,0 +1,101 @@
+.sidebar-container .ps-sidebar-container {
+  background-color: #0e382f;
```

Comment:
We need to avoid having fixed colors and use css variables instead, as we will need to implement light/dark mode feature. Having fixed colors will make this very difficult to implement.

Reply:
done hope you review the new code